docs/azure/sdk/protocol-convenience-methods.md

```diff
@@ -129,6 +129,23 @@ PipelineResponse response = result.GetRawResponse();
 
 ---
 
+## Handle exceptions
+
+When a service call fails, service clients throw an exception that exposes the HTTP status code and the details of the service response, if available.
+
+- `System.ClientModel`-dependent libraries throw a <xref:System.ClientModel.ClientResultException>.
+- `Azure.Core`-dependent libraries throw a <xref:Azure.RequestFailedException>.
+
+# [System.ClientModel exceptions](#tab/system-clientmodel)
+
+:::code source="snippets/protocol-convenience-methods/AzureCore/ExceptionHandling/Program.cs" highlight="10-24":::
+
+# [Azure.Core exceptions](#tab/azure-core)
+
+:::code source="snippets/protocol-convenience-methods/SCM/ExceptionHandling/Program.cs" highlight="9-20":::
```

Comment:
Thinking about this some more, it's probably best to only highlight the catch blocks in these 2 code samples. That is, after all, what we want the customer to focus on.